src/wordlist.rs

```diff
@@ -44,6 +43,8 @@ impl WordList {
     /// This is **WASM-safe** because it doesn’t touch the filesystem —
     /// you can pass the contents of a file fetched via JavaScript `fetch()` or read
     /// from the File API directly into this function.
+    /// That said, we're not using this function in WASM at the moment.
```

Comment:
umm why not? or is it being used indirectly?

Reply:
We currently have vanilla JS do the preprocessing that this function does. I wanted to pass in an array to the main function as opposed to a list of words, to do that pre-processing up front. It is, however, repeated work